runtime-complexity/memoization.md

```diff
@@ -71,17 +44,11 @@ Writing memoization into code can be very simple:-
 
 - when new results are derived check to see if those results are in the hash table;
 
-- if so, call on the results from the hash table to save time
-
-if not, add the new results to the hash table
-
-  
+- if so, call on the results from the hash table to save time, if not, add the new results to the hash table
 
 ### Materials
 
-  
-
--  [Memoization Slideshow](https://docs.google.com/presentation/d/1BipDMgjZd3u-QsrPNCljH-Wv2l3tYRAUz8LWnxzt4s8/edit#slide=id.p)
+  -  [Memoization Slideshow](https://docs.google.com/presentation/d/1BipDMgjZd3u-QsrPNCljH-Wv2l3tYRAUz8LWnxzt4s8/edit#slide=id.p)
 
 -  [Geeks for Geeks](https://www.geeksforgeeks.org/memoization-1d-2d-and-3d/)
```

Comment:
I think the Markdown name of the link could be something else instead of the website name...it could be Memoization (1D, 2D and 3D).

Reply:
Yes, please use descriptive titles for your links. You should also provide a sentence on how to use the resource.